src/GovernorCountingFractional.sol

```diff
@@ -175,3 +178,4 @@ abstract contract GovernorCountingFractional is Governor {
         }
     }
 }
+// forgefmt: disable-stop
```

Comment:
Seems `scopelint fmt` suppresses these warnings from `forge fmt`, I should fix that
```
warning: [src/GovernorCountingFractional.sol:181:1] Invalid inline config item: disable-stop
warning: [src/FractionalPool.sol:260:3] Invalid inline config item: disable-stop
warning: [test/AaveAtokenFork.t.sol:21:1] Invalid inline config item: disable-stop
```

Reply:
Done in bab4ade